docs/ryanjames28.md

```diff
@@ -0,0 +1,7 @@
+# ryanjames28
+
+**Name:** Ryan Bykowski
+
+**Occupation:** Software Engineer
+
+**Location:** Milwaukee Wisconsin
```

Comment:
Hi @ryanjames28 I would suggest adding a comma between the city :beer: and state :cow2: